src/instana/instrumentation/aws/dynamodb.py

```diff
@@ -0,0 +1,40 @@
+# (c) Copyright IBM Corp. 2025
+
+from typing import TYPE_CHECKING, Any, Callable, Dict, Sequence, Type
+
+if TYPE_CHECKING:
+    from botocore.client import BaseClient
+
+from instana.log import logger
+from instana.singletons import tracer
+from instana.span_context import SpanContext
+
+try:
+
+    def collect_dynamodb_attributes(
```

Comment:
I suggest changing the method's name to `create_dynamodb_span()` .

Reply:
Done